fineract-provider/src/main/java/org/apache/fineract/portfolio/account/api/StandingInstructionApiResourceSwagger.java

```diff
@@ -47,7 +47,7 @@ private GetFromOfficeResponseStandingInstructionSwagger() {}
             @Schema(example = "Head Office")
             public String decoratedName;
             @Schema(example = "1")
-            public Integer externalId;
+            public Long externalId;
```

Comment:
THis looks wrong to me based on OfficeData line 37. It should be String.

Reply:
Done.